python/foglamp/services/south/ingest.py

```diff
@@ -560,6 +569,12 @@ def is_available(cls) -> bool:
             if cls._stop:
                 raise RuntimeError('The device server is stopping')
 
+        # Increment the count of received readings to be used for statistics update
+        if asset in cls._sensor_stats:
+            cls._sensor_stats[asset.upper()] += 1
+        else:
+            cls._sensor_stats[asset.upper()] = 1
```

Comment:
in a batch of readings it will be always 1? or  can be n number of for a specific asset sensor

Reply:
This code is inside add_readings which are updated for every read we get, so no point of batch here.